### DISCUSSION TITLE
Module CSS should not be included in the base CSS file

### DISCUSSION THREAD

**Discussion by Hlavtox** [PrestaShop/hummingbird#822] [Ideas]
Module styling should be in separate styles, so they are loaded only when needed. This is how it was done in Prestashop 1.6.

People usually disable many of the default modules, their styling should not be loaded with theme.css.

CCC feature will take care of merging it into 1 file.

**Comment by rodriciru** [comment]
1 filer per module or 1 file for all modules?

**Comment by NeOMakinG** [comment]
Little question, is there a something to check if a module is installed? So we can disable the stylesheet include if the module isn't enabled (because if peoples uninstall a module or disable it, I would like to avoid them to need to remove the stylesheet import from the smarty markup)

**Comment by NeOMakinG** [comment]
And we need to find a way to avoid having a lot of stylesheet link request I think so

**Comment by Hlavtox** [comment]
The stylesheet from `modules/modulename/xxxx.css` or overriden by `themes/theme-refacto/modules/modulename/xxxx.css` will be only loaded, if the module is active and includes it.
